docs/search/extensibility/solr_document_field_mappers.md

```diff
@@ -48,20 +48,15 @@ Each mapper implements two methods, by the same name, but accepting different ar
 Mappers can be used on the extension points by registering them with the [service container](php_api.md#service-container) by using service tags, as follows:
 
 - All block documents
-    - `ContentFieldMapper`
-    - `ibexa.solr.field_mapper.block`
+    - `ibexa.search.solr.field.mapper.block`
 - All block documents per translation
-    - `ContentTranslationFieldMapper`
     - `ibexa.solr.field_mapper.block_translation`
 - Content documents
-    - `ContentFieldMapper`
-    - `ibexa.solr.field_mapper.content`
+    - `ibexa.search.solr.field.mapper.content`
 - Content documents per translation
-    - `ContentTranslationFieldMapper`
-    - `ibexa.solr.field_mapper.content_translation`
+    - `ibexa.search.solr.field.mapper.content.translation`
 - Location documents
-    - `LocationFieldMapper`
-    - `Ibexa\Solr\FieldMapper\LocationFieldMapper\Aggregate`
```

Comment:
I found no trace in the code that tags like `ContentFieldMapper`, `ContentTranslationFieldMapper`, `ContentFieldMapper`, ContentTranslationFieldMapper` or `LocationFieldMapper` could work so I removed them. The others was updated.

Reply:
@alongosz, can you advise on those tags?

Reply:
A also guess `ibexa.solr.field_mapper.block_translation` should be named `ibexa.solr.field_mapper.block.translation` for consistency but that is not the case at the moment....

Reply:
> A also guess `ibexa.solr.field_mapper.block_translation` should be named `ibexa.solr.field_mapper.block.translation` for consistency but that is not the case at the moment....

It's rather consistent within service naming scope. See that next to `@ibexa.solr.field_mapper.block_translation` you have `@ibexa.solr.field_mapper.content_translation`. Looking further there are similar examples like `@ibexa.solr.gateway.endpoint_resolver`. We've discussed service naming within PHP Backend Team and decided to nave not so many groupings. We treat "Field Mapper" and "Block translation" as single entities.

That said, convention around service tags seems to be a bit different. We just look at consistency for service names and tag names separately.

Reply:
> @alongosz, can you advise on those tags?

Seems somehow we used here Service names and IDs instead of service tags. The changes look valid.